src/hb-ot-meta-table.hh

```diff
@@ -62,6 +67,19 @@ struct meta
 {
   static constexpr hb_tag_t tableTag = HB_OT_TAG_meta;
 
+  struct accelerator_t
+  {
+    void init (hb_face_t *face)
+    { table = hb_sanitize_context_t ().reference_table<meta> (face); }
+    void fini () { table.destroy (); }
+
+    hb_blob_t *reference_entry (hb_tag_t tag) const
+    { return table->dataMaps.lsearch (tag, Null (DataMap)).reference_entry (table.get_blob ()); }
```

Comment:
Maybe will be fine with bsearch

Reply:
btw, great to have all these infrastructures...

Reply:
Yep, nice.